### PR TITLE
DISTPG-483 Release notes 15.0

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: docs/release-notes-v15.0.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    * [PostgreSQL 15 release notes](https://www.postgresql.org/docs/15/release-15.html)
+    * Percona Blog: [A Quick Peek at the PostgreSQL 15 Beta 1](https://www.percona.com/blog/a-quick-peek-at-the-postgresql-15-beta-1/)
+    * [PostgreSQL 15: Stats Collector Gone? What’s New?](https://www.percona.com/blog/postgresql-15-stats-collector-gone-whats-new/)
+    
+
+The following is the list of extensions available in Percona Distribution for PostgreSQL.
+
+| Extension           | Version        | Description                  |
+| ------------------- | -------------- | ---------------------------- |
+| [Patroni](https://patroni.readthedocs.io/en/latest/) | 2.1.1 | a HA (High Availability) solution for PostgreSQL |
```
Comment (ImTheKai) [inline docs/release-notes-v15.0.md]: We do ship Patroni version 2.1.4

File: docs/release-notes-v15.0.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+| [pgBackRest](https://pgbackrest.org/)           | 2.40    | a backup and restore solution for PostgreSQL       |
+|[`pgBadger`](https://github.com/darold/pgbadger) | 12.0       | a fast PostgreSQL Log Analyzer.|
+|[`pgBouncer`](https://www.pgbouncer.org/) | 1.17.0 | lightweight connection pooler for PostgreSQL|
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.8   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)| 1.1.1 - Beta2 | collects and aggregates statistics for PostgreSQL and provides histogram information.       |
+| [PostgreSQL Common](https://packages.debian.org/sid/percona-postgresql-common)| 243 | PostgreSQL database-cluster manager. It provides a structure under which multiple versions of PostgreSQL may be installed and/or multiple clusters maintained at one time.|
+|[`wal2json`](https://github.com/eulerto/wal2json) |2.4        | a PostgreSQL logical decoding JSON output plugin.|
```
Comment (ImTheKai) [inline docs/release-notes-v15.0.md]: wal2json is version 2.5

File: docs/release-notes-v15.0.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+|[HAProxy](http://www.haproxy.org/) | 2.5.8 | The high-availability and load balancing solution for PostgreSQL|
```
Comment (ImTheKai) [inline docs/release-notes-v15.0.md]: HAProxy is version 2.5.9

File: docs/release-notes-v15.0.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+| [Patroni](https://patroni.readthedocs.io/en/latest/) | 2.1.1 | a HA (High Availability) solution for PostgreSQL |
+| [pgaudit](https://www.pgaudit.org/)             | 1.7.0 | provides detailed session or object audit logging via the standard logging facility provided by PostgreSQL                |
+|[`pgAudit set user`](https://github.com/pgaudit/set_user)| 3.0.0|  provides an additional layer of logging and control when unprivileged users must escalate themselves to superuser or object owner roles in order to perform needed maintenance tasks.|
+| [pgBackRest](https://pgbackrest.org/)           | 2.40    | a backup and restore solution for PostgreSQL       |
+|[`pgBadger`](https://github.com/darold/pgbadger) | 12.0       | a fast PostgreSQL Log Analyzer.|
+|[`pgBouncer`](https://www.pgbouncer.org/) | 1.17.0 | lightweight connection pooler for PostgreSQL|
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.8   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)| 1.1.1 - Beta2 | collects and aggregates statistics for PostgreSQL and provides histogram information.       |
```
Comment (ImTheKai) [inline docs/release-notes-v15.0.md]: PGSM version is 1.1.1 - no Beta

File: docs/release-notes-v15.0.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    
+
+The following is the list of extensions available in Percona Distribution for PostgreSQL.
+
+| Extension           | Version        | Description                  |
+| ------------------- | -------------- | ---------------------------- |
+| [Patroni](https://patroni.readthedocs.io/en/latest/) | 2.1.1 | a HA (High Availability) solution for PostgreSQL |
+| [pgaudit](https://www.pgaudit.org/)             | 1.7.0 | provides detailed session or object audit logging via the standard logging facility provided by PostgreSQL                |
+|[`pgAudit set user`](https://github.com/pgaudit/set_user)| 3.0.0|  provides an additional layer of logging and control when unprivileged users must escalate themselves to superuser or object owner roles in order to perform needed maintenance tasks.|
+| [pgBackRest](https://pgbackrest.org/)           | 2.40    | a backup and restore solution for PostgreSQL       |
```
Comment (ImTheKai) [inline docs/release-notes-v15.0.md]: pgBackRest is version 2.41

**Review by ImTheKai — CHANGES_REQUESTED** [review]

File: docs/release-notes-v15.0.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+|[`pgBadger`](https://github.com/darold/pgbadger) | 12.0       | a fast PostgreSQL Log Analyzer.|
+|[`pgBouncer`](https://www.pgbouncer.org/) | 1.17.0 | lightweight connection pooler for PostgreSQL|
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.8   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)| 1.1.1 - Beta2 | collects and aggregates statistics for PostgreSQL and provides histogram information.       |
+| [PostgreSQL Common](https://packages.debian.org/sid/percona-postgresql-common)| 243 | PostgreSQL database-cluster manager. It provides a structure under which multiple versions of PostgreSQL may be installed and/or multiple clusters maintained at one time.|
```
Comment (ImTheKai) [inline docs/release-notes-v15.0.md]: postgresql-common version is 241

**Review by umairshahid — CHANGES_REQUESTED** [review]
When we talk about benefits of the distribution, perhaps we should focus on the user problems we are trying to solve. One way is to talk about High Availability and how we recommend addressing it using our reference architecture available here: https://www.percona.com/ha-for-postgresql

**Comment by nastena1606** [comment]
@umairshahid  Could you please elaborate? I'm not sure I understand the request. Do you want me to rewrite the release notes from the benefit point of view?

File: docs/release-notes-v15.0.md (hunk 6 above)
Comment (nastena1606) [reply docs/release-notes-v15.0.md]: Fixed

File: docs/release-notes-v15.0.md (hunk 5 above)
Comment (nastena1606) [reply docs/release-notes-v15.0.md]: Fixed

File: docs/release-notes-v15.0.md (hunk 4 above)
Comment (nastena1606) [reply docs/release-notes-v15.0.md]: Fixed

File: docs/release-notes-v15.0.md (hunk 3 above)
Comment (nastena1606) [reply docs/release-notes-v15.0.md]: Fixed

File: docs/release-notes-v15.0.md (hunk 1 above)
Comment (nastena1606) [reply docs/release-notes-v15.0.md]: fixed

File: docs/release-notes-v15.0.md (hunk 2 above)
Comment (nastena1606) [reply docs/release-notes-v15.0.md]: fixed

**Review by ImTheKai — APPROVED** [review]
LGTM

**Comment by umairshahid** [comment]
@nastena1606 when we describe our distribution, we say " ... aim to simplify deployment and management of your PostgreSQL environment ... ". My suggestion is to talk about user problems that we envision addressing in our PostgreSQL strategy. For example, I might say something like: 

"We are pleased to announce the launch of Percona Distribution for PostgreSQL 15.0 - a clustering solution to assist you in deploying and managing PostgreSQL. The distribution aims to address common operational issues facing enterprises when using PostgreSQL like High Availability, Disaster Recovery, Security, Performance, & Scalability."

Then, in the 'See Also' section, we can add another link to the HA page on our website.

**Comment by nastena1606** [comment]
Thank you for the clarification and ideas.

**Pull request merged**